scripts/deployment/globals_base_mainnet_test.json

```diff
@@ -1 +1 @@
-{"contractVerification":true,"useLedger":false,"derivationPath":"m/44'/60'/2'/0/0","providerName":"base","networkURL":"https://mainnet.base.org","gasPriceInGwei":"0.1","serviceRegistryAddress":"0x3C1fF68f5aa342D296d4DEe4Bb1cACCA912D95fE","karmaAddress":"0x94676070018ff7D09BC32DAc6bb01Ae67282b8E8","karmaProxyAddress":"0x6f7661F52fE1919996d0A4F68D09B344093a349d","fee":"100","minResponseTimeout":"60","maxResponseTimeout":"300","mechMarketplaceAddress":"0xd6AA4Ec948d84f6Db8EEf25104CeE0Ecd280C74e","mechMarketplaceProxyAddress":"0x95dA0F8C3eC5D40209f0EF1ED5E61deD28307d8d","mechFactoryNvmSubscriptionNativeAddress":"0x4cEB52802ef86edF8796632546d89e55c87a0901","drainerAddress":"0xE49CB081e8d96920C38aA7AB90cb0294ab4Bc8EA","wrappedNativeTokenAddress":"0x4200000000000000000000000000000000000006","tokenCreditRatio":"990000000000000000","balanceTrackerNvmSubscriptionNativeAddress":"0xaC26774616bbeD41b0CB69EA2ae7de366F430b23","mechFactoryFixedPriceNativeAddress":"0x0338893fB1A1D9Df03F72CC53D8f786487d3D03E","mechFactoryFixedPriceTokenAddress":"0x1d333b46dB6e8FFd271b6C2D2B254868BD9A2dbd","balanceTrackerFixedPriceNativeAddress":"0x6d9b08701Af43D68D991c074A27E4d90Af7f2276","balanceTrackerFixedPriceTokenAddress":"0x3c3007a2e4ba7f73C764F69f6D6C1983D9EB6B26","subscriptionNFTAddress":"","subscriptionTokenId":""}
```

Comment:
The configuration file now has test setup contracts just to test all the static audits.